common/build/eslint-config-fluid/src/custom-rules/no-restricted-tags-imports.js

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	meta: {
+		type: "problem",
+		docs: {
+			description:
+				"This rule restricts imports from internal or non-public APIs. This to prevent accidental dependencies on internal, unstable or undocumented parts of the codebase.",
```

Comment:
We'll want to update this to be a bit more general, since this rule will work for any tag.